build/run-in-docker

```diff
@@ -63,10 +63,10 @@ $DOCKER_CMD run $DOCKER_GPU_OPTS $DOCKER_RUN_EXTRA_ARGS -u $(id -u):$(id -g) --r
   -v "/etc/passwd:/etc/passwd:ro" \
   -v "/etc/shadow:/etc/shadow:ro" \
   -v "/etc/sudoers.d:/etc/sudoers.d:ro" \
-  -v "$REPODIR:$REPODIR:rw" \
+  -v "$WORKDIR:$WORKDIR:rw" \
```

Comment:
Oops missed that WORKDIR is not set in this script by default to REPODIR like it was in the previous revision.  Does not work if WORKDIR is not set at all.